bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsResultsAnalyzer.scala

```diff
@@ -125,14 +125,14 @@ object GemsResultsAnalyzer {
         }
       }
       val gemsGuideStars = go(Nil, TiptiltFlexurePair.pairs(catalogSearch))
-      sortResultsByRanking(gemsGuideStars.toList)
+      sortResultsByRanking(gemsGuideStars)
     }
   }
 
   private def printResults(result: List[GemsGuideStars]) {
-    Log.info("Results:")
+    Log.fine("Results:")
     result.zipWithIndex.foreach { case (s, i) =>
-      Log.info(s"result #$i : $s")
+      Log.fine(s"result #$i : $s")
```

Comment:
Right, this is very old stuff so I'm not sure if we should just remove it altogether